purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -684,19 +694,10 @@ internal class BillingWrapper(
                     log(LogIntent.GOOGLE_WARNING, message)
                     // The calls will fail with an error that will be surfaced. We want to surface these errors
                     // Can't call executePendingRequests because it will not do anything since it checks for isReady()
-                    synchronized(this@BillingWrapper) {
-                        while (true) {
-                            serviceRequests.poll()?.let { serviceRequest ->
-                                mainHandler.post {
-                                    serviceRequest(
-                                        billingResult.responseCode
-                                            .billingResponseToPurchasesError(message)
-                                            .also { errorLog(it) },
```

Comment:
Hmm I guess before we would have created a new error and logged it for every `serviceRequest`... but the new code seems better, so I'm ok with the change 👍

Reply:
Yes, I was meaning to mention the change. I agree it looks better now